types/option.go

```diff
@@ -233,6 +233,9 @@ type PutObjectOptions struct {
 	TxnHash          string // TxnHash indicates the transaction hash creating the object meta on chain.
 	DisableResumable bool   // DisableResumable indicates whether upload the object to Storage Provider via resumable upload.
 	PartSize         uint64
+	Delegated        bool
```

Comment:
It would better to have some comments here to help developers understand the relationship between the "Delegated" field value and ToggleSPAsDelegatedAgent tx.